### PR TITLE
Add plan review for Telemetry 2.0

### PR DESCRIPTION
**Pull request by Emily-Jiang**

### CODE REVIEW

File: telemetry/2.0/README.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: "MicroProfile Telemetry 2.0"
+date: 2024-05-03
+summary: 
+---
+MicroProfile Telemetry is to adopt OpenTelemetry from CNCF to allow MicroProfile Applications benefit from OpenTelemetry and enable Jakarta RESTful Web Services automatically being traced if configured.
```
Comment (Emily-Jiang) [inline telemetry/2.0/README.adoc:6]:
```suggestion
MicroProfile Telemetry is to adopt OpenTelemetry from CNCF to allow MicroProfile Applications benefit from OpenTelemetry and enable Jakarta RESTful Web Services automatically being observed if configured.
```

File: telemetry/2.0/README.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+title: "MicroProfile Telemetry 2.0"
+date: 2024-05-03
+summary: 
+---
+MicroProfile Telemetry is to adopt OpenTelemetry from CNCF to allow MicroProfile Applications benefit from OpenTelemetry and enable Jakarta RESTful Web Services automatically to be observed if configured.
+
+This is a major release. This release adopts the OpenTelemetry Metrics and Logs in addition to the OpenTelemetry Tracing in the previous MicroProfile Telemetry releases.
+The following updates are provided:
+
+* Adopt the latest release of OpenTelemetry
```
Comment (Emily-Jiang) [inline telemetry/2.0/README.adoc:11]:
```suggestion
* Adopt the latest release of OpenTelemetry 
* Adopt the latest release of Semantic Conventions 
```

File: telemetry/2.0/README.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+title: "MicroProfile Telemetry 2.0"
+date: 2024-05-03
```
Comment (Emily-Jiang) [inline telemetry/2.0/README.adoc:3]:
```suggestion
date: 2024-06-19
```

File: telemetry/2.0/README.adoc [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+title: "MicroProfile Telemetry 2.0"
+date: 2024-06-19
+summary: 
+---
+MicroProfile Telemetry is to adopt OpenTelemetry from CNCF to allow MicroProfile Applications benefit from OpenTelemetry and enable Jakarta RESTful Web Services automatically to be observed if configured.
```
Comment (jclingan) [inline telemetry/2.0/README.adoc:6]: My recommended verbiage in italics:

MicroProfile Telemetry adopts OpenTelemetry from the CNCF and adds MicroProfile specification integration for improved MicroProfile application observability.

Comment (Emily-Jiang) [inline telemetry/2.0/README.adoc:6]:
```suggestion
MicroProfile Telemetry adopts OpenTelemetry from the CNCF and adds MicroProfile specification integration for improved MicroProfile application observability.
```

**Review by jclingan — APPROVED** [review]
Looks good.

File: telemetry/2.0/README.adoc [hunk 5]
```diff
@@ -0,0 +1,34 @@
+---
+title: "MicroProfile Telemetry 2.0"
+date: 2024-06-19
+summary: 
+---
+MicroProfile Telemetry adopts OpenTelemetry from the CNCF and adds MicroProfile specification integration for improved MicroProfile application observability.
```
Comment (emecas) [inline telemetry/2.0/README.adoc:6]: MicroProfile Telemetry adopts https://opentelemetry.io/[OpenTelemetry] from the https://www.cncf.io/projects/opentelemetry/[CNCF] and adds MicroProfile specification integration for improved MicroProfile application observability.

**Review by emecas — APPROVED** [review]
LGTM

I added an optional suggestion

Thank you @Emily-Jiang

**Review by brunobat — APPROVED** [review]

**Pull request merged**